src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall(Ball ball) {
```

Comment:
```suggestion
    public static Ball getRandomBall() {
```
ми можемо одразу в методі створити об'єкт класу Ball щоб не приймати пустий об'єкт ззовні